### ISSUE TITLE
Material icons should be accessible

### ISSUE THREAD

**Issue by marcysutton** [angular/material#427]
To be accessible, `<md-icon>` needs two things:
1. `aria-label` should be enforced. (see #344)
2. The nested `<object>` tag needs `wmode="opaque"` or `wmode="transparent"` since it is for presentation only ([Flash a11y best practices](https://www.webaccessibility.com/best_practices.php?technology_platform_id=51))

I remember something about a refactor possibly forthcoming, I just wanted to capture these requirements.
